locales/config.yaml

```diff
@@ -24,3 +24,6 @@ gettext:
   # content, relative to the project root directory
   exclude_files:
     - 'lib/puppet/pops/types/type_formatter.rb'
+    # The semantic_puppet gem is temporarily vendored (PUP-7114), and it
+    # handles its own translation.
+    - 'lib/semantic_puppet/*.rb'
```

Comment:
Can you use globs here?  We have both `lib/semantic_puppet.rb` and `lib/semantic_puppet/**/*` I believe.